src/main/java/chess/db/DBConnector.java

```diff
@@ -0,0 +1,31 @@
+package chess.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBConnector {
+
+    private static final String URL = "jdbc:mysql://localhost:3306/chess";
+    private static final String USER = "user";
+    private static final String PASSWORD = "password";
+
+    public static Connection getConnection() {
+        Connection connection = null;
+        roadDriver();
+        try {
+            connection = DriverManager.getConnection(URL, USER, PASSWORD);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
```

Comment:
```suggestion
        roadDriver();
        try {
           return DriverManager.getConnection(URL, USER, PASSWORD);
        } catch (SQLException e) {
            throw ...
        }
```
이런식으로하면 불필요한 null할당을 줄일 수 있을 것 같아요.
또 여기 catch부분에서도 `e.printStackTrace();`만 하도록되어있는데, 이 부분도 보완하면 좋겠습니다.